packages/ckeditor5-dev-build-tools/src/plugins/translations.ts

```diff
@@ -14,7 +14,7 @@ import type { Plugin } from 'rollup';
 import { removeWhitespace } from '../utils';
 
 const TYPINGS = removeWhitespace( `
-	import type { Translations } from 'ckeditor5';
+	import type { Translations } from '@ckeditor/ckeditor5-utils';
```

Comment:
As it's a bug fix, we should have a test for it.

Reply:
The `Fix` was the best commit type I could think of, but maybe `Other` is actually better. There's no bug in the current implementation - this change is just so that packages have `@ckeditor/ckeditor5-utils` as a dependency, not the entire `ckeditor5'.

IMO testing this doesn't make much sense, as the test would have to have a copy of this string and check if it matches.

Reply:
If so, I would check if types are not imported from `ckeditor5`, and that would be enough in case somebody wants to update it in the future.

Reply:
I added a new test for translations typings. :heavy_check_mark: